doc/2/controllers/security/search-users/snippets/search-users-koncorde.dart

```diff
@@ -0,0 +1,18 @@
+final result = await kuzzle.security.searchUsers(query: {
+  'query': {
```

Comment:
Same here, why we have 2 times the `query` property?

Reply:
As we discussed, this is normal here